apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.js

```diff
@@ -641,16 +641,19 @@ type OperationDetailsExtraProps = {
 };
 
 const OperationDetailsExtra = ({ extra }: OperationDetailsExtraProps) => {
-  return Object.entries(extra).map(([key, value]) => (
-    <OpDetailsSection key={key}>
-      <OpDetailsTitle>
-        <Trans i18nKey={`operationDetails.extra.${key}`} defaults={key} />
-      </OpDetailsTitle>
-      <OpDetailsData>
-        <Ellipsis>{value}</Ellipsis>
-      </OpDetailsData>
-    </OpDetailsSection>
-  ));
+  return Object.entries(extra).map(([key, value]) => {
```

Comment:
generally, changes in the generic part must be shipped in a separate PR as a coin integration should only touch its "family". but to be check with others if it's acceptable for this one.